simple_network_sim/network_of_populations/__init__.py

```diff
@@ -279,9 +278,10 @@ def _distributeContactsOverAges(ageToSusceptibles, totalSusceptibles, newContact
     :rtype: A dictionary of ages (keys) and the number of new infections (values)
     """
     if stochastic:
-        assert isinstance(newContacts, int)
+        assert isinstance(newContacts, int) or newContacts.is_integer()
 
         ageProbabilities = np.array(list(ageToSusceptibles.values())) / totalSusceptibles
+        print(ageProbabilities)
```

Comment:
You probably want to leave out this print or convert it to a logger.debug

Reply:
oh well catched, totally forgot to remove this one